### DISCUSSION TITLE
[@types/node] AbortSignal type misses `timeout` method

### DISCUSSION THREAD

**Discussion by atassis** [DefinitelyTyped/DefinitelyTyped#61877] [Issues with a @types package]
Description of a method from [MDN](https://developer.mozilla.org/en-US/docs/Web/API/AbortSignal/timeout)

[a comment by an automated account is not shown]

**Reply by atassis** [reply]
How does it work? Not the biggest issue- no response whatsoever.

**Comment by atassis** [comment]
Implemented [this](https://github.com/DefinitelyTyped/DefinitelyTyped/compare/master...atassis:DefinitelyTyped:patch-1), but haven't read all the requirements in PR description for making it, so staling the commit for now

**Comment by peterblazejewicz** [comment]
This is native feature, if being added to @types/node, needs to be aligned with TS features:
https://github.com/microsoft/TypeScript/issues/48003
